WarOfEmpires.QueryHandlers.Tests/Empires/Worker.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace WarOfEmpires.QueryHandlers.Tests.Empires {
+    internal class Worker {
+    }
```

Comment:
Is this class used anywhere? There are no references to it

Reply:
Huh I have no idea how that got there.